internal/pkg/agent/application/info/agent_metadata.go

```diff
@@ -199,6 +201,16 @@ func (i *AgentInfo) ECSMetadataFlatMap() (map[string]interface{}, error) {
 	}
 
 	info := sysInfo.Info()
+	hostname := info.Hostname
+	l.Debugf("in ECSMetadataFlatMap, hostname = %s", hostname)
```

Comment:
I only added this for debugging purposes, so we could see debug-level logs from this method regardless of whether the FQDN feature flag was set or not.  Will remove before this PR is merged.